bundle/jsky.app.ot/src/main/java/jsky/app/ot/viewer/DBProgramChooserFilter.java

```diff
@@ -183,25 +170,21 @@ private static void restoreSettings(List<JCheckBox> checks, boolean def) {
         for (final JCheckBox c : checks) {
             final String pref = PREF_KEY + "." + c.getName();
             c.setSelected(Preferences.get(pref, def));
-            c.addActionListener(new ActionListener() {
-                public void actionPerformed(ActionEvent e) {
-                    Preferences.set(pref, Boolean.toString(c.isSelected()));
-                }
-            });
+            c.addActionListener(e -> Preferences.set(pref, Boolean.toString(c.isSelected())));
         }
     }
 
     private void restoreSettings() {
-        restoreSettings(Arrays.asList(remote, classical, lp, fastTurn, queue), true);
-        restoreSettings(Arrays.asList(cal, engineering, other, libs),         false);
+        restoreSettings(Arrays.asList(remote, classical, lp, fastTurn, queue, other), true);
+        restoreSettings(Arrays.asList(cal, engineering, libs),         false);
```

Comment:
This is the "meat" of the PR, if you will: `other` programs are now displayed by default, whereas before, they were hidden by default.